tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -3040,6 +3040,20 @@ def test_resolved_alert_updates_metric_issue(self, mock_produce_occurrence_to_ka
         assert status_change.new_status == GroupStatus.RESOLVED
         assert occurrence.fingerprint == status_change.fingerprint
 
+    @with_feature("organizations:workflow-engine-metric-alert-processing")
+    @mock.patch("sentry.incidents.subscription_processor.process_data_packets")
+    def test_process_data_packets_called(self, mock_process_data_packets):
+        rule = self.rule
+        detector = self.create_detector(name="hojicha", type="metric_alert_fire")
```

Comment:
recommend importing the `grouptype.slug` here so you won't need to update it if the text changes, and CI will fail if there's a mismatch in the future.